wheatley/row_generation/place_notation_generator.py

```diff
@@ -55,7 +57,11 @@ def summary_string(self) -> str:
         return f"place notation '{self.method_pn_string}'"
 
     def _gen_row(self, previous_row: Row, stroke: Stroke, index: int) -> Row:
-        lead_index = index % self.lead_len
+        lead_index = (index + self.start_index) % self.lead_len
+        # Correct for the fact that Python's modulus is objectively **DUMB** when you feed it negative
+        # numbers.  In Python, -10 % 3 = -1 not 2 as we need
```

Comment:
Is this true on your machine?
It's documented as having the same sign as the second operand which is the behaviour I get
https://docs.python.org/3/library/functions.html#divmod
```
> -10 % 3
2
> 10 % -3
-2
```

Reply:
Oooh I didn't know that - in which case it's not objectively dumb and I take the comment back :laughing:.  Also that means the `if` statement will never execute, so doesn't need to exist...